test.js

```diff
@@ -1 +1,11 @@
-console.log('Hello World')
+console.log('Hello World');
+
+console.log('Hello Universe');
+
+console.log('Hello other stuff');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/troubleshoot-codacy/no-suggestions-2/pullRequest?prid=8692776)
```suggestion
console.log("Hello other stuff");
```